src/python/foglamp/storage/storage.py

```diff
@@ -129,7 +122,35 @@ def check_shutdown(self):
         conn.close()
         return json.loads(res)
 
+    def get_storage_service(self):
+        """ get Storage service """
+
+        # TODO: URL to service registry api?
+        conn = http.client.HTTPConnection("localhost:8082")
```

Comment:
This will come from the command line arguments eventually

Reply:
Yupe with FOGL-517 and start up fix. Hence a TODO.